.drone.yml

```diff
@@ -35,7 +35,7 @@ steps:
     dockerfile: package/Dockerfile
     password:
       from_secret: docker_password
-    repo: "rancher/harvester"
+    repo: "harvester/harvester"
```

Comment:
The docker repo is not changed due to rate limiting, so we don't need to change it.